### PR TITLE
added home page with category

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added Home Page With Category.
-> Used Tailwind Css
-> Added Data of the questions

### CODE REVIEW

File: src/Pages/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Category } from "../../Data/Category";
+import { useNavigate } from "react-router-dom";
+
+export const Home = () => {
+  console.log(Category);
```
Comment (deepak29-git) [inline src/Pages/Home/Home.js]: remove console

**Review by deepak29-git — APPROVED** [review]

File: src/Pages/InCategory/InCategory.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+      <div className="text-2xl font-medium mb-5 text-center">
+        {curr_quiz_obj.category_name}
+      </div>
+      <div className="w-11/12 md:w-9/12 mx-auto md:flex md:gap-8">
+        {curr_quiz_obj.quizzes.map((ele) => (
+          <div
+            key={ele._id}
+            className="bg-slate-700 rounded-3xl mb-8 flex-1 cursor-pointer"
+            // onClick={() => {
```
Comment (Prabhav-Sharma) [inline src/Pages/InCategory/InCategory.js]: Remove comments, if not needed in future development

**Review by Prabhav-Sharma — APPROVED** [review]

File: src/Pages/InCategory/InCategory.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            //   navigate(`/in-category/${ele._id}`);
+            // }}
+          >
```
Comment (prathmeshjagtap) [inline src/Pages/InCategory/InCategory.js]: you can remove commented code if not required

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean and Good 
Just remove commented code and Console.logs

**Review by tanishk15sharma — APPROVED** [review]
good Naming convention .

File: src/Pages/Home/Home.js (hunk 1 above)
Comment (ashishnnnnn) [reply src/Pages/Home/Home.js]: Done...

File: src/Pages/InCategory/InCategory.js (hunk 2 above)
Comment (ashishnnnnn) [reply src/Pages/InCategory/InCategory.js]: Done..

File: src/Pages/InCategory/InCategory.js (hunk 3 above)
Comment (ashishnnnnn) [reply src/Pages/InCategory/InCategory.js]: Done..

**Pull request merged**